src/ObjectProperty/KeyColumn.js

```diff
@@ -45,20 +53,6 @@ class KeyColumn extends React.Component {
       },
     };
 
-    /*
-    const isPropertyReferenced = (
-      (
-        this.context.includedProperties !== 'show' && this.context.inheritedProperties !== 'show'
-      ) && (
-        this.context.includedProperties !== 'tag' && this.context.inheritedProperties !== 'tag'
-      )
-    );
-
-    if (isPropertyReferenced) {
-      style.keyColumn.paddingLeft = '20px';
-    }
-    */
```

Comment:
Also ignore the last comment regarding this, I suppose 😛 It's hard to review commit-by-commit!